README.md

```diff
@@ -586,3 +586,20 @@ $ DEBUG=nemo:error <nemo command>
 Because we NEed MOre automation testing!
 
 [![NPM](https://nodei.co/npm/nemo.png?downloads=true&stars=true)](https://nodei.co/npm/nemo/)
+
+## Unit Tests
+
+* Unit tests run by default using headless browser [PhantomJS](http://phantomjs.org/). To run unit tests out of box, You must have PhantomJS installed on your system and must be present in the path
+    * Download PhantomJS from [here](http://phantomjs.org/download.html)
+    * On OSX, you can optionally use `brew` to install PhantomJS like `brew install phantomjs`
+    * PhantomJS installation detailed guide on Ubuntu can be found [here](https://gist.github.com/julionc/7476620)
+
+* If you want to run unit tests on your local browser, like lets say Firefox/Chrome (make sure ChromeDriver is in current path), you need to update browser in unit test
+configuration, for example the browser section under `test/config/config.json` like [here](https://github.com/paypal/nemo/blob/master/test/config/config.json#L19)
+
+* How to run unit tests?
+  * `grunt simplemocha` will just run unit tests
+  * `grunt` - default grunt task will run linting as well as unit tests
```

Comment:
I could be wrong but currently npm test is mapped to grunt (default) via package.json scripts. How bout mapping it to grunt simple mocha and npm run lint to grunt. That way we can recommend using npm test and npm run lint.